xprof/tally.cpp

```diff
@@ -213,14 +214,17 @@ static void ust_thapi_metadata_usr_callback(void *btx_handle, void *usr_data,
   data->metadata.push_back(metadata);
 }
 
+
 void btx_register_usr_callbacks(void *btx_handle) {
   btx_register_callbacks_initialize_usr_data(btx_handle, &btx_initialize_usr_data);
   btx_register_callbacks_read_params(btx_handle, &btx_read_params);
   btx_register_callbacks_finalize_usr_data(btx_handle, &btx_finalize_usr_data);
 
-  btx_register_callbacks_lttng_host(btx_handle, &host_usr_callback);
-  btx_register_callbacks_lttng_device(btx_handle, &device_usr_callback);
-  btx_register_callbacks_lttng_traffic(btx_handle, &traffic_usr_callback);
   btx_register_callbacks_lttng_device_name(btx_handle, &device_name_usr_callback);
```

Comment:
Why is this one called `lttng_device_name`?

Reply:
Stupid Historical reason. Will be fixed, when we will migrate all the interval to metababel

Reply:
Sounds good!